scalyr_agent/scalyr_client.py

```diff
@@ -660,8 +660,9 @@ def __receive_response(self, body_str, send_time):
                         error_code="client/requestFailed",
                     )
                 else:
-                    log.exception(
+                    log.warning(
                         "Failed to receive response due to exception.  Closing connection, will re-attempt",
+                        exc_info=True,
```

Comment:
Yes, as you said it yourself - we definitely want to include original stack trace here to provide more context on what is going on.